### ISSUE TITLE
Using absolute path restricts what you can do a lot

### ISSUE THREAD

**Issue by robclancy** [oreporan/color-tabs-vscode#21]
I just want to set it to folders, but I can't because `app` is in another folder so it overrides.

**Comment by oreporan** [comment]
@robclancy 

I'm not sure I understand. You're using absolute paths as part of your regex?

Can you paste your configuration here?

Something like:

```
 "colorTabs.config": [
    {
      "regex": ".*/web/.*",
      "color": "#c0d8e3",
      "label": "WEB"
    },
    {
      "regex": ".*/mobile/.*",
      "color": "#e18a7a",
      "label": "MOBILE"
    }
  ]
```

**Comment by robclancy** [comment]
No, you are using absolute paths to check the regex against.

**Comment by oreporan** [comment]
I understand now. 
You're right. I will push a fix to start only at the root of the workspace

Good catch

**Comment by oreporan** [comment]
fixed in #22

**Comment by robclancy** [comment]
Ty! <3

**Comment by oreporan** [comment]
Released in 0.7.1
